layers/+lang/zig/packages.el

```diff
@@ -20,7 +20,7 @@
     :defer t
     :init
     (progn
-      ;; config goes here.
+      (spacemacs/add-to-hook 'zig-mode-hook '(spacemacs//zig-setup-backend))
```

Comment:
If you just add one function to one hook you should consider using the standard add-to-hook function from emacs. Also please pass only the function in there, using the `#'` form, this helps the compiler to detect function passing.